observability/metrics/src/main/scala/com/daml/metrics/Timed.scala

```diff
@@ -74,6 +75,13 @@ object Timed {
     future.andThen { case _ => counter.dec() }(ExecutionContext.parasitic)
   }
 
+  def trackedFuture[T](meter: Meter, future: => Future[T])(implicit context: MetricsContext) = {
+    meter.mark()(context.withLabels(("state", "started")))
```

Comment:
Do we want to use labels or different meters? Same way we did for gRPC requests for example.

Reply:
As this is the first metric that uses this approach, I'd suggest to not introduce a generalized metric yet but use different meters for now.